src/components/Head/index.js

```diff
@@ -1,13 +1,16 @@
 import Helmet from "react-helmet";
 import PropTypes from "prop-types";
 import React from "react";
+import { path } from "ramda"
 
 // --------------------------------------------------
 
 import { theme, } from "../../styles";
 
 
 const Head = ( { site, page, } ) => {
+	const image = path(['image', 'url'])(page) || path(['homeImage', 'url'])(site)
```

Comment:
```suggestion
	const image = R.path(['image', 'url'])(page) || R.path(['homeImage', 'url'])(site)
```

Reply:
I can see myself getting minorly confused at some future date without the `R.` lol